lib/discourse_theme/cli_commands/rspec.rb

```diff
@@ -88,7 +88,7 @@ def run_tests_locally(local_directory, spec_path, headless: false)
 
           Kernel.exec(
             ENV,
-            "cd #{local_directory} && #{headless ? SELENIUM_HEADLESS_ENV : ""} bundle exec rspec #{spec_path}",
+            "cd #{local_directory} && #{!headless ? SELENIUM_HEADLESS_ENV : ""} bundle exec rspec #{spec_path}",
```

Comment:
```suggestion
            "cd #{local_directory} && #{headless ? SELENIUM_HEADLESS_ENV : ""} bundle exec rspec #{spec_path}",
```

Reply:
Yup I need to rename `SELENIUM_HEADLESS_ENV` to `SELENIUM_HEADFUL_ENV` as well because what the env does is to enable headful mode.